lib/cosmos/system/target.rb

```diff
@@ -186,7 +186,8 @@ def add_all_cmd_tlm(dir)
       if Dir.exist?(File.join(dir, 'cmd_tlm'))
         # Only grab *.txt files in the root of the cmd_tlm folder
         Dir[File.join(dir, 'cmd_tlm', '*.txt')].each do |filename|
-          cmd_tlm_files << filename
+          # Don't add partial files which begin with '_'
+          cmd_tlm_files << filename unless filename[0] == '_'
```

Comment:
The partials need to stay in the cmd_tlm_files list so that they will be included in the MD5 calculation.   You just need to modify System#load_packets to not call process_file on them.  Not sure if that should be based on .erb extension.   See higher level comment